lib/datasets/rdatasets.rb

```diff
@@ -0,0 +1,126 @@
+require_relative "dataset"
+require_relative "tar_gz_readable"
+
+module Datasets
+  class RDatasets < Dataset
```

Comment:
It seems that `Rdatasets` (`d` not `D`) is the official name.

```suggestion
  class Rdatasets < Dataset
```

Reply:
OK, I'll rename this class name.